x/assets/types/genesis.go

```diff
@@ -42,13 +42,23 @@ func (gs GenesisState) Validate() error {
 				i,
 			)
 		}
+		// this is our primary method of cross-chain communication.
 		if info.LayerZeroChainID == 0 {
```

Comment:
can we consider setting the default values for **LayerZeroChainID** and **AddressLength** to avoid the 0 case.

Reply:
As discussed, this is the client chain's address length and each chain may have different length for their addresses. It should be supplied accurately in the genesis file.